server/db/models/product.js

```diff
@@ -0,0 +1,57 @@
+const Sequelize = require('sequelize')
+const db = require('../index')
+
+
+const Product = db.define('product', {
+    name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: false
+        }
+    },
+    description: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+        validate: {
+            notEmpty: false
+        }
+    },
+    ingredients: {
+        type: Sequelize.ARRAY(Sequelize.STRING),
+        allowNull: false,
+        validate: {
+            notEmpty: false
+        }
+    },
+    price: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        validate: {
+            notEmpty: false
+        }
+    },
+    timeToPrep: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        validate: {
+            notEmpty: false
+        }
+    },
+    availabiliy: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        defaultValue: false
+    },
+    averageRating:  {
```

Comment:
i believe average rating should not be hard coded, and we can use an aggregate sql function, like average